README.md

```diff
@@ -1,3 +1,81 @@
 # loopback-cli
 
 LoopBack CLI tool for creating projects, models and more.
+This package supersedes the older `slc` tool.
+
+## Installation
+
+```
+$ npm install loopback-cli
+```
+
+## Usage
+
+### Getting started
+
+ 1. Run `lb` to create a new LoopBack application
```

Comment:
`lb app` is already working as expected. `lb` is an alias for `lb app`, similarly to how `yo loopback` is an alias for `yo loopback:app`.

I am happy to revisit this alias, please open a new issue to discuss.